injected/src/url-change.js

```diff
@@ -39,13 +48,15 @@ function listenForURLChanges() {
     const historyMethodProxy = new DDGProxy(urlChangedInstance, History.prototype, 'pushState', {
         apply(target, thisArg, args) {
             const changeResult = DDGReflect.apply(target, thisArg, args);
-            handleURLChange();
+            console.log('pushstate event');
+            handleURLChange('pushState');
```

Comment:
iiuc `State` isn't a suffix on the navigation event `navigationType` field.

```suggestion
            handleURLChange('push');
```